core/services/aws/tests/user/cognitoTrigger/lambdas/postConfirmationTrigger.test.ts

```diff
@@ -1,56 +1,128 @@
 import postConfirmationLambda from '../../../../user/cognitoTrigger/lambdas/postConfirmationTrigger'
 import { UserService } from '../../../../../../application/userWorkflows/UserService'
 import DynamoDbTableOperations from '../../../../commons/ddb/DynamoDbTableOperations'
-import { postConfirmationLambdaMockEvent } from '../../../cannedData/lambdaEventMocks'
 import { mockDynamoDbOperations } from '../../../mock/dynamoDbMocks'
 import { mockUserWithStringId } from '../../../../../../application/tests/mocks/mockUserData'
 import { updateCognitoUserInfo } from '../../../../commons/cognito/CognitoOperations'
+import { sendAppUserWellcomeMail } from '../../../../commons/ses/sesOperations'
+import { GenericMessage } from '../../../../../../../commons/dto/MessageDTO'
+import { createPostConfirmationEvent } from '../../../mock/cognitoEventMocks'
 
 jest.mock('../../../../../../application/userWorkflows/UserService')
 jest.mock('../../../../commons/ddb/DynamoDbTableOperations')
 jest.mock('../../../../commons/cognito/CognitoOperations')
+jest.mock('../../../../commons/ses/sesOperations')
 
 describe('postConfirmationLambda Tests', () => {
   const mockDynamoDbTableOperations = mockDynamoDbOperations
+  const mockEmailContent: GenericMessage = {
+    title: 'Welcome',
+    content: 'Welcome content'
+  }
 
   beforeEach(() => {
     (DynamoDbTableOperations as jest.Mock).mockImplementation(() => mockDynamoDbTableOperations)
     jest.spyOn(UserService.prototype, 'createNewUser').mockResolvedValue(mockUserWithStringId)
+    jest.spyOn(UserService.prototype, 'getAppUserWellcomeMail').mockReturnValue(mockEmailContent)
+    ;(updateCognitoUserInfo as jest.Mock).mockResolvedValue(undefined)
+    ;(sendAppUserWellcomeMail as jest.Mock).mockResolvedValue(undefined)
```

Comment:
Why `;` is in front of the line.

Reply:
fixed it at 00e5082f21b0937112b0accc54ff1e1a25f1ee7f